tests/node/fixtures/yarn-v2-workspaces/.yarnrc.yml

```diff
@@ -0,0 +1,3 @@
+nodeLinker: node-modules
+
+yarnPath: .yarn/releases/yarn-3.2.0.cjs
```

Comment:
@JGAntunes Shipping a binary for a specific version of yarn might be controversial, but
1. it's a valid real-world use case (we use this configuration at work)
2. it's faster to run the tests, because we don't have to download a new yarn version.